### PR TITLE
Be able to use a Document as input not only its name (needed to manage multiple document)

### PR DESCRIPTION
**Pull request by pabloalonso**

### CODE REVIEW

**Comment by pozil** [comment]
+1 on that one, I have the same need

File: bonita-connector-cmis-common/pom.xml [hunk 1]
```diff
@@ -9,7 +9,7 @@
 	</parent>
 
 	<artifactId>bonita-connector-cmis-common</artifactId>
-	<version>2.0.1</version>
+	<version>2.1.0</version>
```
Comment (rbioteau) [inline bonita-connector-cmis-common/pom.xml]: should't be a minor update ?

**Comment by rbioteau** [comment]
I do not understand the realtion between mutliple document and this PR. Can you provide some additional information please.

**Comment by pozil** [comment]
Current connector only takes this input for the document to upload:
`String documentName`

This implementation only support single Bonita documents but it does not allow to upload document lists thus the PR.

**Comment by rbioteau** [comment]
Following PR https://github.com/bonitasoft/bonita-connector-cmis/pull/4 handle the use case.

**Pull request closed without merging**